src/site/content/en/blog/fetch-metadata/index.md

```diff
@@ -123,32 +135,41 @@ To ensure that your site can still be linked from other sites, you have to allow
       return True  # Allow this request
 ```
 {% Aside 'gotchas' %}
-The logic above protects your application's endpoints from being used as resources by other websites, but will permit top-level navigation and embedding (loading in an <iframe>). To further improve security, you can use Fetch Metadata headers to restrict cross-site navigations to only an allowed set of pages.
+The logic above protects your application's endpoints from being used as resources by other websites, but will permit top-level navigation and embedding (e.g. loading in an <iframe>). To further improve security, you can use Fetch Metadata headers to restrict cross-site navigations to only an allowed set of pages.
 {% endAside %} 
 
 
-#### Step 4: (Optional) Opt out endpoints that are meant to serve cross-site traffic
+#### Step 4: Opt out endpoints that are meant to serve cross-site traffic (Optional) 
+
 In some cases, your application might provide resources which are meant to be loaded cross-site. These resources need to be exempted on a per-path or per-endpoint basis. Examples of such endpoints are:
- - CORS-enabled endpoints
- - Public resources like stylesheets or images
+
+ - Endpoints meant to be accessed cross-origin - If your application is serving endpoints that are `CORS` enabled, you need to explicitly opt them out from resource isolation to ensure that cross-site requests to these endpoints are still possible.
+ - Public resources (e.g. images, styles, etc) - Any public and unauthenticated resources that should be loadable cross-origin from other sites can be exempted as well.
+
+```python
+  if req.path in ('/my_CORS_endpoint', '/favicon.png'):
+    return True
```

Comment:
```suggestion
if req.path in ('/my_CORS_endpoint', '/favicon.png'):
  return True
```

Reply:
Can you please remove the indentation from all code examples? It's causing the page to render with the wrong left-hand spacing.